docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -242,11 +308,13 @@ Standard TensorFlow\* operations:
 | SquaredDifference | No |
 | Square| No |
 | Squeeze | The case when squeeze axis is not specified is not supported |
+| StatelessWhile | No |
 | StopGradient | Not needed for shape inference |
 | StridedSlice | Supported only for constant-foldable begin, end, and strides inputs |
 | Sub | No |
 | Sum | No |
 | Swish | No |
+| swish_f32 | No |
 | Switch | Control flow propagation |
 | Tan | No |
 | Tanh | No |
```

Comment:
could you please add TensorListReserve, TensorListSetItem, TensorListGetItem, TensorListFromTensor, TensorListStack? "Supported only when it is part of a sub-graph of the special form"

Reply:
@lazarevevgeny asked not to include TensorList* operations, because they are kind of internal operations.

Reply:
@lazarevevgeny can you comment?

Reply:
I am ok to proceed without these operations. Sorry, I missed your agreement.